mmdet/datasets/coco.py

```diff
@@ -427,6 +427,8 @@ def evaluate_det_segm(self,
         if iou_thrs is None:
             iou_thrs = np.linspace(
                 .5, 0.95, int(np.round((0.95 - .5) / .05)) + 1, endpoint=True)
+        else:
+            iou_thrs = np.array(iou_thrs)
```

Comment:
We can assert whether `iou_thrs` is a float or a list. If is a float, we can set `iou_thrs= [iou_thrs]`.